packages/earth-admin/src/pages-client/layers/details.tsx

```diff
@@ -14,133 +14,476 @@
+                                  isSearchable
+                                  placeholder="Select layer provider"
+                                  styles={CUSTOM_STYLES}
+                                  theme={theme => ({
+                                    ...theme,
+                                    ...SELECT_THEME,
+                                  })}
+                                  rules={{required: true}}/>
+                    </div>
+                    <div>
+                      <label htmlFor="type">Layer type</label>
+                      <Controller as={Select} control={control} className="marapp-qa-type"
+                                  name="type"
+                                  options={LAYER_TYPE_OPTIONS}
+                                  defaultValue={layerType}
+                                  isSearchable
+                                  placeholder="Select layer type"
+                                  styles={CUSTOM_STYLES}
+                                  theme={theme => ({
+                                    ...theme,
+                                    ...SELECT_THEME,
+                                  })}
+                                  rules={{required: true}}/>
+                    </div>
+                  </>
+                )}>
+                <div className="ng-margin-medium-bottom">
+                  <p className="ng-text-weight-bold ng-margin-remove">Layer provider</p>
+                  <p className="ng-margin-remove ng-padding-left">{provider}</p>
+                </div>
+                <div>
+                  <p className="ng-text-weight-bold ng-margin-remove">Later type</p>
+                  <p className="ng-margin-remove ng-padding-left">{type}</p>
+                </div>
+              </InlineEditCard>
+            </div>
+          </div>
+          <div className="ng-grid">
+            <div className="ng-width-1-1">
+              <InlineEditCard
+                onSubmit={onSubmit}
+                validForm={true}
+                render={({setIsEditing, setIsLoading, setServerErrors}) => (
+                  <>
 
-  return (
-    <ContentLayout errors={errors} backTo="/layers" isLoading={isLoading} className="marapp-qa-layersedit">
-      <LayerEdit data={data} newLayer={path.newLayer}/>
+                    <div className="ng-margin-medium-bottom">
+                      <label htmlFor="config">Layer Config</label>
+                      <Controller
+                        name="config"
+                        control={control}
+                        defaultValue={layerConfig}
+                        onChange={(layerConfig) => handleJsonChange(layerConfig)}
+                        as={<JsonEditor json={layerConfig}/>}
+                      />
+                    </div>
+
+                  </>
+                )}>
+                <div className="ng-margin-medium-bottom">
+                  {config && <div>
```

Comment:
Just to make it more readable
```suggestion
                  {config && (
                      <div>
                      ...
                      </div>
                   )}
```